firebase-ai/src/test/java/com/google/firebase/ai/DevAPIUnarySnapshotTests.kt

```diff
@@ -95,4 +98,22 @@ internal class DevAPIUnarySnapshotTests {
     goldenDevAPIUnaryFile("unary-failure-unknown-model.json", HttpStatusCode.NotFound) {
       withTimeout(testTimeout) { shouldThrow<ServerException> { model.generateContent("prompt") } }
     }
+
+  // This test case can be removed once b/422779395 is
```

Comment:
Just making sure that this won't be visible to the developer in the ref docs?

Reply:
Yes this won't be visible to the user in the ref docs. Anything that isn't in the `/** */` style comments are not visible in the reference docs.